### ISSUE TITLE
Unable to creating deeper subfolders in Vod folder name field

### ISSUE THREAD

**Issue by olig89** [MrBrax/LiveStreamDVR#472]
When using `\` in the vod folder field, it shows the desired behaviour in the preview but removes the character and combines the two strings either side in reality. 

![image](https://github.com/MrBrax/LiveStreamDVR/assets/24813487/d6caed2e-20f4-44cf-9c5b-ae746185034e)

![image](https://github.com/MrBrax/LiveStreamDVR/assets/24813487/1cc36cbd-2cd6-4c26-9430-e97618473e69)

For my usecase, I can probably get away with not having the "season" sub-folders if this functionality is a pain but it would make sense for the preview to reflect the outcome.

**Comment by MrBrax** [comment]
this is indeed untested behaviour, as slashes probably break the path calculation

**Comment by olig89** [comment]
So for background (and for anyone that reads this in future / wants a similar outcome) my full usecase is basically to auto-download vods from a couple of variety streamers I sub to who regularly try out new games and, if the game vibes well, will develop into a medium-long playthrough. I'm unable to keep up with them live, and I find twitch's vod player a pain to use for this purpose, forever forgetting my position in a video, and manually downloading whole episodes for offline viewing forces you to 3rd party apps/sites anyway.

As it's not time critical I use the (imho) less resource hungry "Download VOD at end of capture/stream end" setting in LivestreamDVR.

### Using Plex as a VOD server

The reason for the weird Season & Episode naming is because I use Plex as my viewer and that naming format works well both for breaking it up into manageable chunks and keeping it automatically organised natively in Plex like any other TV Series.

Until now, I had it setup like:
```
- Streams (The Plex Library, the "vods" folder LivestreamDVR creates)
   - Streamer (Plex: TV Show name, LivestreamDVR's streamer subfolder)
      - Season (Plex: year and month in 4 digits, LivestreamDVR's "Vod folder name")
         - Episodes (Plex: same season, and day & hour, LivestreamDVR's "Vod filename")
```

e.g.
![image](https://github.com/MrBrax/LiveStreamDVR/assets/24813487/20304edd-3780-48a8-9fa4-888099e34e27)

resulting in: 
![image](https://github.com/MrBrax/LiveStreamDVR/assets/24813487/c1a0cc08-9f8f-4400-b040-8a68b8a8de87)

The "issue" I faced after a few months was it became quite annoying to parse through when the streamer either plays something I don't care about or they play two good things but I want to watch them out of sequence (i.e. latest one first).

### New approach
So for now I'll use the {game_name} as the show title and shift the Plex libraries to go down a level to be:

```
- Streamer (The Plex Library, LivestreamDVR's streamer subfolders)
   - Gamename (Plex: TV Show name, LivestreamDVR's "Vod folder name")
      <no additional season folders>
         - Episodes (Plex: yr, mth, day & hour, LivestreamDVR's "Vod filename")
```

Using Plex's ability to treat the whole show as a single season, results in:
![image](https://github.com/MrBrax/LiveStreamDVR/assets/24813487/dc0e6d94-da46-4e5e-89ca-ebccd658b013)

If you follow a lot of streamers the number of libraries may make the Plex UX a bit gross, but seems ok for a few.

The only issue I can see cropping up for me is if they stream multiple games in a single session - I don't yet know how to split it up the processed video based on chapters.

Feel free to edit/copy this elsewhere if it helps.

**Comment by MrBrax** [comment]
you can use [lossless-cut](https://github.com/mifi/lossless-cut/) with the provided file next to the video to extract segments, but i don't think it's generated for download-only

**Comment by olig89** [comment]
> you can use [lossless-cut](https://github.com/mifi/lossless-cut/) with the provided file next to the video to extract segments, ~but i don't think it's generated for download-only~

Looks good, will investigate but on first glance it doesnt look like there's a native way to use something like a watch/import folder to trigger an action/automation.

**Comment by olig89** [comment]
Also, looks like I need season folders after all, Plex can only have episodes up to 4 digits - gameplay may cross a calendar month change but I need both day&hour to cater for multiple streams per day.

Alternatively, I see there's `{episode}` field in LivestreamDVR but can't get it to resolve through to the filenames, what does it relate to?

**Comment by MrBrax** [comment]
season and episodes automatically get filled in when capturing streams, probably not for downloaded ones. it would most likely solve this issue as it rolls over for every month. could use some fixing up.

and for the splitting, could also be implemented in some kind of automation i suppose

**Comment by olig89** [comment]
Let me know if you need help with testing either the subdirectory creation with `/`'s in the folder field, or the seasons & eps resolving for downloads.

**Comment by MrBrax** [comment]
yeah both of them will be available when the latest commit is built

**Comment by MrBrax** [comment]
very neat. next commit has new template options for the new split by chapters button in advanced vod controls, with customisable names

**Comment by MrBrax** [comment]
so i've been thinking, maybe it would be best to remove the use of channel folders and instead just make the vod folder template include it by default and that it can be changed if the user wishes

**Comment by olig89** [comment]
So you mean remove the standalone checkbox option, and only have that option in the vod folder bit?

It's a more logical place for it, and keep things together for better UX so 👍 from me
